src/v2/Apps/Home/Components/HomeFeaturedMarketNews.tsx

```diff
@@ -20,22 +20,31 @@ import {
 import { compact, take } from "lodash"
 import * as React from "react";
 import { createFragmentContainer, graphql } from "react-relay"
+import { StyledLink } from "v2/Components/Links/StyledLink";
```

Comment:
We don't need to import this. What you want to style is actually the parent anchor tag, which in this case is a `RouterLink`

Reply:
Incorporated the `RouterLink` instead.